src/Microsoft.Identity.Client/AppConfig/IApplicationConfiguration.cs

```diff
@@ -29,7 +29,12 @@ namespace Microsoft.Identity.Client.AppConfig
 {
     internal interface IApplicationConfiguration : IAppConfig
     {
+        /// <summary>
+        /// ExtendedLifeTimeEnabled is a Boolean that first party applications (read Office) can set to true in case when the STS has an outage,
```

Comment:
>first party applications [](start = 54, length = 24)

should we mention first party here? is this open to anyone? I would modify this to be more generic

Reply:
This is currently an internal interface, so I think it's ok to talk about first party applications only here for now.  The IsExtendedTokenLIfetimeEnabled isn't supposed to be public (yet)

---
In reply to: [248454890](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/791#discussion_r248454890) [](ancestors = 248454890)

Reply:
sure..good point.

---
In reply to: [248471012](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/791#discussion_r248471012) [](ancestors = 248471012,248454890)

Reply:
updating this.  talked with henrik offline


---
In reply to: [248471187](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/791#discussion_r248471187) [](ancestors = 248471187,248471012,248454890)